### PR TITLE
temp branch

### PR DESCRIPTION
**Pull request by jhhocs**

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+            }
+            //Validates and asks for eligible meal fee based on departure time
+            if (eligibleDepartMeals(departureTime) == 1)
+            {
+                printf("You're eligible for a covered breakfast! Enter the cost of your breakfast: ");
+                scanf("%f", &departMeal);
+            }
+            else if (eligibleDepartMeals(departureTime) == 2)
+            {
+                printf("You're eligible for a covered lunch! Enter the cost of your breakfast: ");
```
Comment (jhhocs) [inline main.c:68]: typo, breakfast should be lunch

File: main.c [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+                scanf("%f", &departMeal);
+            }
+            else if (eligibleDepartMeals(departureTime) == 3)
+            {
+                printf("You're eligible for a covered dinner! Enter the cost of your breakfast: ");
```
Comment (jhhocs) [inline main.c:73]: typo, breakfast should be dinner

**Review by katherine-co — APPROVED** [review]
fixed typos

File: main.c [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+    {
+        fflush(stdin);
+        printf("\nWould you like to use this program:\nPress 1 if yes\nPress 2 to quit the program\n");
+        scanf("%d", &choice);
+        
+        if(choice == 1)
+        {
+            //Gets user's total number of days on trip 
+            printf("How many days was your trip? ");
+            scanf("%d", &totalNumOfDays);
```
Comment (jhhocs) [inline main.c:45]: Add fflush(stdin) after every scanf call

File: main.c [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+                printf("Sorry, you are not eligible for a covered meal.\n");
+            }    
+
+            //Calculates covered meal costs for first and last day
+            mealFees(eligibleDepartMeals(departureTime), eligibleArrivalMeals(arrivalTime), departMeal, arriveMeal);   //double check if can pass function in function
+
+            //Asks user if rented car was used
+            printf("\nDid you rent a car during your trip? Enter 1 for true and 0 for false: ");
+            scanf("%d", &carRented);
+            while(carRented < 0 || carRented > 1)
```
Comment (jhhocs) [inline main.c]: change to while(carRented != 0 && carRented != 1)

File: FeesCalculator.h [hunk 5]
```diff
@@ -0,0 +1,18 @@
+double allowedExpenses;
```
Comment (jhhocs) [inline FeesCalculator.h]: Delete "double allowedExpenses;"

File: main.c (hunk 4 above)
Comment (jhhocs) [reply main.c]: bool carRented can be changed to int carRented

**Review by katherine-co — APPROVED** [review]
Deleted "double allowedExpenses" from .h file

**Review by katherine-co — APPROVED** [review]
Added fflush(stdin)

**Review by katherine-co — APPROVED** [review]
changed booleans to int to take in user input

**Pull request merged**